prov/rxm/src/rxm_conn.c

```diff
@@ -484,6 +489,27 @@ void rxm_cmap_process_conn_notify(struct rxm_cmap *cmap,
 	rxm_conn_connected_handler(handle);
 }
 
+static inline void rxm_cmap_progress_wait_conn_quota_queue(struct rxm_cmap *cmap)
```

Comment:
Maybe just name this rxm_cmap_progress_conn_wait_list?  That's still a little long, but rxm_cmap_progress_wait_conn_quota_queue is almost a full paragraph.  :)